README.md

```diff
@@ -29,7 +29,7 @@ A source is getting associated with a consumer group ID the first time the `Read
 | name                 | description                                                                                                                                                                                                  | required | default value             |
 |----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------------------|
 | `servers`            | Servers is a list of Kafka bootstrap servers, which will be used to discover all the servers in a cluster.                                                                                                   | true     |                           |
-| `topic`              | Topic is the Kafka topic from which records will be read.                                                                                                                                                    | true     |                           |
+| `topic`              | Topic is a comma separated list of Kafka topics from which records will be read, ex: "topic1,topic1".                                                                                                        | true     |                           |
```

Comment:
We should include `topics` here and add a mention regarding deprecating the use of `topic`. This connector is currently on 0.7.2. Maybe we could say that will be deprecated by 1.0.

Reply:
oh sorry forgot about the readme! updated